.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "16:00"
+      timezone: "Europe/London"
+    commit-msg:
+      prefix: "fix(deps)"
+    labels:
+      - "dependencies"
+    open-pull-requests-limit: 3
+    reviewers:
+      - "Rust-Scientific-Computing/Developers"    
```

Comment:
Is it case-agnostic? Otherwise it should be `Rust-Scientific-Computing/developers`.

Reply:
It is "as if you were @ mentioning the group". So @Rust-Scientific-Computing/developers is right. Good catch!